pkg/kubelet/server/stats/types.go

```diff
@@ -104,6 +95,10 @@ type PodReference struct {
 
 // NetworkStats contains data about network resources.
 type NetworkStats struct {
+	// The time at which these stats were updated.
+	Time unversioned.Time `json:"time"`
+	// The time at which data collection for the cumulative stats was (re)started.
+	StartTime unversioned.Time `json:"startTime"`
```

Comment:
We will need `StartTime` for the container and pod though, to provide uptime metrics.